packages/ui/cypress/tests/watched-accounts.cy.ts

```diff
@@ -58,4 +63,154 @@ describe('Watched Accounts', () => {
+      accountDisplay.pureBadge().should('not.exist')
+    })
+    // navigate to the home page and ensure the multisig badge is included in the header
+    topMenuItems.homeButton().click()
+    multisigPage.accountHeader().within(() => {
+      accountDisplay.identicon().should('be.visible')
+      accountDisplay.multisigBadge().should('be.visible')
+      accountDisplay.pureBadge().should('not.exist')
+    })
+  })
+
+  it('can see the identicon and badge displayed for a watched pure proxy', () => {
+    cy.visit(settingsPageWatchAccountUrl)
+    addWatchAccount(
+      watchMultisigs['multisig-with-pure'].pureAddress,
+      watchMultisigs['multisig-with-pure'].name)
+    // ensure the pure badge is displayed in the selectable drop-down option
+    topMenuItems.multiproxySelector()
+      .should('be.visible')
+      .first().click()
+    topMenuItems.multiproxySelectorOption().within(() => {
+      accountDisplay.identicon().should('be.visible')
+      accountDisplay.pureBadge().should('be.visible')
+      accountDisplay.multisigBadge().should('not.exist')
+    })
+    // navigate to the home page and ensure the pure badge is included in the header
+    topMenuItems.homeButton().click()
+    multisigPage.accountHeader().within(() => {
+      accountDisplay.identicon().should('be.visible')
+      accountDisplay.pureBadge().should('be.visible')
+      accountDisplay.multisigBadge().should('not.exist')
+    })
+  })
+
+  it('can edit the name of a watched pure', () => {
+    cy.visit(settingsPageWatchAccountUrl)
+    addWatchAccount(
+      watchMultisigs['multisig-with-pure'].pureAddress,
+      watchMultisigs['multisig-with-pure'].name)
+    // navigate to the home page and edit the name
+    topMenuItems.homeButton().click()
+    multisigPage.optionsMenuButton().click()
+    multisigPage.editNamesMenuOption().click()
+    editNamesModal.body().should('be.visible')
+    editNamesModal.inputEditPureName().type(`{selectall}{del}${`Edited Name Test`}`)
+    cy.wait(500)
+    editNamesModal.saveButton()
+      .should('be.enabled')
+      .click()
+    // ensure the edited name is now displayed in the home page header
+    multisigPage.accountHeader().within(() => {
+      accountDisplay.nameLabel()
+        .should('have.text', 'Edited Name Test')
+    })
+    // navigate to settings and ensure the edited name is displayed
+    cy.visit(settingsPageWatchAccountUrl)
+    settingsPage.accountContainer().within(() => {
+      accountDisplay.nameLabel()
+        .should('have.text', 'Edited Name Test')
+    })
+  })
+
+  it('can see a subscan link for a watched pure', () => {
+    cy.visit(settingsPageWatchAccountUrl)
+    addWatchAccount(
+      watchMultisigs['multisig-with-pure'].pureAddress,
+      watchMultisigs['multisig-with-pure'].name)
+    // navigate to the home page and edit the name
+    topMenuItems.homeButton().click()
+    multisigPage.optionsMenuButton().click()
+    multisigPage.subscanMenuOption().should('be.visible')
+  })
```

Comment:
This is a very basic test atm, do we want to try and remove the target on the link so that the tab loads in the same window and then check that the subscan URL contains the correct address....or is that too much?

Reply:
I imagine that you meant to write this for the subscan test. I agree that it'd be better to actually test that the link is for the right address at least. I didn't research it but I guess it's possible. Removing the target sounds like hack though :shrug:

Reply:
Updated: @Tbaut Juan has a nice solution for checking external links on another FE project. The same technique works well for us

Reply:
yup that's great, I figured there were some ways to stub this but didn't check :+1: